changelogs/fragments/strip_keys_fixes.yml

```diff
@@ -0,0 +1,3 @@
+bugfixes:
+    - change function to in place replacement, compose with module_args_cpoy for 'new clean copy'
```

Comment:
should `module_args_cpoy` be `module_args_copy` or is `_cpoy` an acronym I'm not familiar with?